docs/xml/nginx/changes.xml

```diff
@@ -5,6 +5,64 @@
 <change_log title="nginx">
 
 
+<changes ver="1.27.2" date="2024-10-02">
+
+<change type="feature">
+<para lang="ru">
+оптимизация использования памяти и уменьшение времени запуска
+в конфигурациях с большим количеством SSL-контекстов, в которых
+директивы ssl_certificate, ssl_client_certificate,
+ssl_trusted_certificate и ssl_crl указывают на одни и те же файлы.
+</para>
+<para lang="en">
+memory usage optimization and startup speedup
+in configurations with a large number of SSL contexts with the same
+paths specified by the "ssl_certificate", "ssl_client_certificate",
+"ssl_trusted_certificate", and "ssl_crl"  directives.
```

Comment:
This gives impression that the caching is implemented only for the specified directives.
Consider adding something like "and equivalent directives of the proxy, grpc and uwsgi modules" or removing explicit list of supported directives to resolve ambiguity.

Reply:
That's true. I will change back to my previous version with a list of "SSL certificates, secret keys, and CRL lists". Although this may not look as clear, listing all possible directives is way too long.